packages/gen-typescript-declarations/README.md

```diff
@@ -54,28 +62,27 @@ process:
 $ npm install --save-dev @polymer/gen-typescript-declarations
 ```
 
-Add a `gen-tsd` script to your `package.json`:
+Add a `generate-types` script to your `package.json`:
 
 ```js
 {
   ...
   "scripts": {
-    "gen-tsd": "gen-tsd"
+    "generate-types": "gen-typescript-declarations"
   }
 }
 ```
 
-If you're using Bower, ensure you run `npm run gen-tsd` to generate the latest
-typings and commit them to your repository before tagging each release.
+If you're using NPM, you can add this script to the NPM `prepack` script to
+generate and include typings in your NPM package every time you publish.
 
-If you're using NPM, you can instead add this script to the NPM `prepack`
-script to generate and include typings in your NPM package every time you
-publish.
+If you are still using Bower, ensure you run `npm run gen-tsd` to generate the
+latest typings and commit them to your repository before tagging each release.
```

Comment:
Might want to mention that if they add typings to .gitignore they need to have an .npmignore that does not include the typings, in order for them to be published. We might also want to recommend that they add the typings to gitignore since they're autogenerated.

Reply:
Done.